lib/mocha/integration/minitest/adapter.rb

```diff
@@ -44,7 +44,7 @@ def before_teardown
         # @private
         def after_teardown
           super
-          mocha_teardown
+          Mockery.teardown("#{self.class.name}##{name}")
```

Comment:
I'm not as familiar with the `minitest` codebase as I once was, but doesn't it already provide a way to access an identifying string like this? Maybe something like [`Minitest::Runnable#name`](https://www.rubydoc.info/gems/minitest/Minitest/Runnable#name-instance_method), i.e. could we change your implementation to `Mockery.teardown(name)`? (I haven't actually tried this, but something like this would feel better in that it would use the test library's string representation of a test.

Reply:
Minitest has no such thing. `name` only return the test method name. e.g. `FakeTest#test_thing`, `name` returns `"test_thing"`.

Reply:
You can see Minitest doing this internally: https://github.com/minitest/minitest/blob/287b35d60c8e19c11ba090efc6eeb225325a8520/lib/minitest.rb#L551